dist/pom.xml

```diff
@@ -135,6 +135,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>net.sourceforge.javacsv</groupId>
+            <artifactId>javacsv</artifactId>
+            <version>${javacsv.version}</version>
+            <scope>provided</scope>
```

Comment:
I thought the goal was to include the `javacsv` jar with the zep package.  Wouldn't the `provided` mean that the `javacsv` jar will _not_ be included the final package?  The that it'll assume that the JDK/JRE or the container will provide that jar file instead?

Reply:
Nope, 
<img width="621" alt="image" src="https://github.com/user-attachments/assets/50d43786-e047-4202-b2e6-edf294320d91" />
Behaviour would be the same as for Janino jar, which is included in the lib dir in the screen above

Reply:
It's true for a general build with the Maven build plugin and fat Jars, but we have a separate module dist with an assembly XML specification of what jars we need to include and into what directory
<img width="1477" alt="image" src="https://github.com/user-attachments/assets/fbc7b5f5-9572-49a0-8332-b255e5fb597d" />
The jar is included, you can check that with this job of the current branch 
http://platform-jenkins.zenoss.eng/job/Components/job/zenoss-zep/job/branch/1/console